recaf-ui/src/main/java/me/coley/recaf/ui/jfxbuilder/component/attribute/CSSStyleable.java

```diff
@@ -0,0 +1,5 @@
+package me.coley.recaf.ui.jfxbuilder.component.attribute;
+
+public interface CSSStyleable<Self extends CSSStyleable<Self>> {
+	Self styleSheet(String stylePath);
```

Comment:
Currently it is assumed all components are already defined as styles in the css files. Then all they need to do is add the relevant classes. All windows inherit the same CSS sheets so everything _"just works"_ rather than having to specify sheets when needed.